config/constants.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
+  MAPBOX_API_TOKEN:
+    'pk.eyJ1IjoieWNob3kiLCJhIjoiY2pmOTYwdzZ5MG52dDJ3b2JycXY4ZDU5ciJ9.m9H_Mqu1b42AObg_u_tjpA',
```

Comment:
Do we want this in the repo? might need a config file

Reply:
@JMStudiosJoe No, I don't think we want this in exposed in the repo. What do you mean by config file? For me, I would keep this info as a local environment variable.

Reply:
Environment variable is one way another is to have a config.json file that is ignored from hit and imported when needed to access the key along with any other keys